src/wh_server_crypto.c

```diff
@@ -697,7 +697,7 @@ static int _HandleEccVerify(whServerContext* ctx, whPacket* packet,
     byte* res_pub       = (uint8_t*)(res + 1);
     word32 max_size     = (word32)(WOLFHSM_CFG_COMM_DATA_LEN -
                                 (res_pub - (uint8_t*)packet));
-    uint16_t pub_size   = 0;
+    uint32_t pub_size   = 0;
```

Comment:
```suggestion
   word32 pub_size   = 0;
```
Best to match the wc type here

Reply:
Match which wolfCrypt size? It isn't being passed to any wolfCrypt API and it being assigned to `res->pubSize`, which is a `uint32_t`? The size is returned from wolfCrypt as an `int` too, which gets stored in `ret`, then assigned to `pubSize`.